app/javascript/components/forms/profile/styles.scss

```diff
@@ -9,7 +9,7 @@
 
   h3 {
     font-size: rem(16px);
-    margin-bottom: rem(40px);
+    margin-bottom: rem(28px);
```

Comment:
Reason for change?

Reply:
The top margin I added to the fields. Rolling back.